tasks/framework-tools/src/frameworkDepsToProject.js

```diff
@@ -41,6 +41,6 @@ try {
 
   console.log('... Done. Now run `yarn install`')
 } catch (e) {
-  console.log('Error:', e)
+  console.log(e)
```

Comment:
I like the `Error` part.

Reply:
I took it out because it was redundant when I threw a specific package json syntax error (here's what it looks like with it in—it's `Error: Error: `)

![image](https://user-images.githubusercontent.com/32992335/125118875-4fabcf80-e0be-11eb-8a67-a8f0e178ccf4.png)

Reply:
Not sure if I did it correctly yet just noticed the redundancy; it'd be better if the Error I threw didn't include Error so that other Errors still had the prefix. I'll see if I can make it happen